service-front/module/Application/src/Helpers/LocalisationHelper.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Helpers;
+
+use Application\Exceptions\LocalisationException;
+
+class LocalisationHelper
+{
+    public function __construct(private readonly array $config)
+    {
+    }
+
+    private array $wordMap = [
+        'PASSPORT' => "Passport",
+        'DRIVING_LICENCE' => 'Driving licence',
+        'NATIONAL_ID' => 'National ID',
+        'RESIDENCE_PERMIT' => 'Residence permit',
+        'TRAVEL_DOCUMENT' => 'Travel document',
+    ];
+
+    /**
+     * @throws LocalisationException
+     */
+    public function getInternationalSupportedDocuments(array $detailsData): array
```

Comment:
To avoid a dependency on the shape of the case data, could this just take `countryCode` as an argument?
```suggestion
    public function getInternationalSupportedDocuments(string $countryCode): array
```